lib/danger/helpers/comments_helper.rb

```diff
@@ -151,17 +152,25 @@ def random_compliment
          "Yay.", "Jolly good show.", "Good on 'ya.", "Nice work."].sample
       end
 
-      def character_from_emoji(emoji)
+      def character_from_emoji(emoji, template: "github")
         emoji.delete! ":"
-        if emoji == "no_entry_sign"
-          "🚫"
-        elsif emoji == "warning"
-          "⚠️"
-        elsif emoji == "book"
-          "📖"
-        elsif emoji == "white_check_mark"
-          "✅"
-        end
+        
+        container = {
+          "github" => {
+            "no_entry_sign"    => "🚫",
+            "warning"          => "⚠️",
+            "book"             => "📖",
+            "white_check_mark" => "✅"
+          },
+          "bitbucket_server" => {
+            "no_entry_sign"    => "\u274C", # ❌
+            "warning"          => "\u2757", # ❗
+            "book"             => "\u2728", # ✨
+            "white_check_mark" => "\u2705"  # ✅
+          }
+        }
+
+        return container[template][emoji]
       end
```

Comment:
What do you think about extract the logic here to a class?

```ruby
def character_from_emoji(emoji, template: "github")
  WarningEmoji.new(emoji, template).call
end
```

```ruby
class WarningEmoji
  DATA = {
    "github" => {
      "no_entry_sign"    => "🚫",
      "warning"          => "⚠️",
      "book"             => "📖",
      "white_check_mark" => "✅"
    },
    "bitbucket_server" => {
      "no_entry_sign"    => "\u274C", # ❌
      "warning"          => "\u2757", # ❗
      "book"             => "\u2728", # ✨
      "white_check_mark" => "\u2705"  # ✅
    }
  }

  def initialize(emoji, template)
    @emoji = emoji.delete ":"
    @template = template
  end

  def call
    DATA[template][emoji]
  end

  private

    attr_reader :emoji, :template
end
```

I am bad at naming things, you can change to what suits the best :)